airbyte-webapp-e2e-tests/cypress/integration/source.spec.js

```diff
@@ -1,4 +1,8 @@
 describe("Source main actions", () => {
+  beforeEach(() => {
```

Comment:
This is done to always start fresh from the new workspace, right?

Reply:
Yes that fakes the response of the workspaces to act like they have been through the initial setup flow. Otherwise the app would show the preference screen for those tests. This worked only so far because the onboarding test suit ran before this one and worked through that screen. But that requires those test suits to run in a specific dependent order, which we should not rely on (and recording is actually breaking that order). With faking this response slightly, we can make sure that tests won't run into the preference screen (or run into it, if needed), as we require.